### PR TITLE
As a potential customer I can see a navbar with the links to all the sections of the website, so that I can reach them quickly

### PR DESCRIPTION
**Pull request by lspk129**
https://tree.taiga.io/project/troublemaker-website/us/3?milestone=135017

### CODE REVIEW

File: components/Navbar/NavbarSection/index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+const NavbarSection = styled.nav`
+  width: 100%;
+  background-color: ${props => props.theme.main};
+  overflow: hidden;
+  display: grid;
+  grid-template-rows: 1fr;
+  z-index: 10;
```
Comment (tmxyz) [inline components/Navbar/NavbarSection/index.js]: change to 1

File: components/Navbar/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+          </Scroll.Link>
+          <RaisedButton
+            label="Contact us"
+            labelColor={props.muiTheme.palette.accent1Color}
+            labelStyle={{ fontSize: '15px' }}
+            onClick={() => Scroll.animateScroll.scrollToBottom()}
+          />
+        </NavbarList>
+      </NavbarSection>
+        )}
```
Comment (tmxyz) [inline components/Navbar/index.js]: fix the indentation

File: components/Navbar/index.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  </Sticky>
+);
+
+Navbar.propTypes = {
+  muiTheme: PropTypes.shape({
+    palette: PropTypes.object,
```
Comment (tmxyz) [inline components/Navbar/index.js]: isRequired

File: components/Navbar/index.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    accent1Color: PropTypes.string,
```
Comment (tmxyz) [inline components/Navbar/index.js]: isRequired

File: components/Navbar/NavbarLogo/index.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import NavbarSection from '../NavbarSection';
+import NavbarImage from './NavbarImage';
+import NavbarImageWrapper from './NavbarImageWrapper';
+
+const NavbarLogo = () => (
+  <NavbarSection>
+    <NavbarImageWrapper>
+      <NavbarImage src="../../static/logo_main.png" alt="moka" />
```
Comment (lspk129) [inline components/Navbar/NavbarLogo/index.js]: Fixed.

File: components/Navbar/NavbarSection/index.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+const NavbarSection = styled.nav`
+  width: 100%;
+  background-color: ${props => props.theme.main};
+  overflow: hidden;
```
Comment (lspk129) [inline components/Navbar/NavbarSection/index.js]: Fixed.

File: components/Navbar/NavbarSection/index.js (hunk 1 above)
Comment (lspk129) [inline components/Navbar/NavbarSection/index.js]: Fixed.

File: components/Navbar/index.js (hunk 2 above)
Comment (lspk129) [inline components/Navbar/index.js]: Fixed.

File: components/Navbar/index.js (hunk 3 above)
Comment (lspk129) [inline components/Navbar/index.js]: Fixed.

File: components/Navbar/index.js (hunk 4 above)
Comment (lspk129) [inline components/Navbar/index.js]: Fixed.

**Pull request merged**